_monitoring-plugins/ad/api.md

```diff
@@ -668,1043 +1929,1016 @@ POST _plugins/_anomaly_detection/detectors/results/_search
+          ]
+        },
+        ...
+      ],
+      "ad_canceled_batch_task_count": 0,
+      "ad_hc_execute_request_count": 0,
+      "ad_hc_execute_failure_count": 0,
+      "model_count": 22,
+      "ad_execute_failure_count": 0,
+      "ad_batch_task_failure_count": 0,
+      "ad_total_batch_task_execution_count": 28,
+      "ad_executing_batch_task_count": 3
+    }
   }
 }
 ```
 
+The `model_count` parameter shows the total number of models running on each node’s memory.
+For historical analysis, you see the values for the following fields:
 
-Use `job=true` to get anomaly detection job information.
+- `ad_total_batch_task_execution_count`
+- `ad_executing_batch_task_count`
+- `ad_canceled_batch_task_count`
+- `ad_batch_task_failure_count`
+
+If haven't run any historical analysis, these values show up as 0.
+
+To get all stats for a specific node:
 
 #### Request
 
 ```json
-GET _plugins/_anomaly_detection/detectors/<detectorId>?job=true
+GET _plugins/_anomaly_detection/<nodeId>/stats
+```
+
+To get specific stats for a node:
+
+#### Request
+
+```json
+GET _plugins/_anomaly_detection/<nodeId>/stats/<stat>
+GET _plugins/_anomaly_detection/<nodeId>/stats/ad_execute_request_count
 ```
 
 #### Sample response
 
 ```json
 {
-  "_id" : "m4ccEnIBTXsGi3mvMt9p",
-  "_version" : 1,
-  "_primary_term" : 1,
-  "_seq_no" : 3,
-  "anomaly_detector" : {
-    "name" : "test-detector",
-    "description" : "Test detector",
-    "time_field" : "timestamp",
-    "indices" : [
-      "order*"
-    ],
-    "filter_query" : {
-      "bool" : {
-        "filter" : [
-          {
-            "exists" : {
-              "field" : "value",
-              "boost" : 1.0
-            }
-          }
-        ],
-        "adjust_pure_negative" : true,
-        "boost" : 1.0
-      }
-    },
-    "detection_interval" : {
-      "period" : {
-        "interval" : 1,
-        "unit" : "Minutes"
-      }
-    },
-    "window_delay" : {
-      "period" : {
-        "interval" : 1,
-        "unit" : "Minutes"
-      }
-    },
-    "schema_version" : 0,
-    "feature_attributes" : [
-      {
-        "feature_id" : "mYccEnIBTXsGi3mvMd8_",
-        "feature_name" : "total_order",
-        "feature_enabled" : true,
-        "aggregation_query" : {
-          "total_order" : {
-            "sum" : {
-              "field" : "value"
-            }
-          }
-        }
-      }
-    ],
-    "last_update_time" : 1589441737319
-  },
-  "anomaly_detector_job" : {
-    "name" : "m4ccEnIBTXsGi3mvMt9p",
-    "schedule" : {
-      "interval" : {
-        "start_time" : 1589442051271,
-        "period" : 1,
-        "unit" : "Minutes"
-      }
-    },
-    "window_delay" : {
-      "period" : {
-        "interval" : 1,
-        "unit" : "Minutes"
-      }
-    },
-    "enabled" : true,
-    "enabled_time" : 1589442051271,
-    "last_update_time" : 1589442051271,
-    "lock_duration_seconds" : 60
+  "nodes": {
+    "SWD7ihu9TaaW1zKwFZNVNg": {
+      "ad_execute_request_count": 12
```

Comment:
How about add more description about this sample response that it's only to get `ad_execute_request_count` on node "SWD7ihu9TaaW1zKwFZNVNg".